app/lib/linters/crunch_api/fetch.rb

```diff
@@ -0,0 +1,29 @@
+require 'rest-client'
+require 'retries'
+require 'base64'
+module Linters
+
+  module CrunchApi
+    CRUNCH_BASE_URL = ENV.fetch('CRUNCH_BASE_URL', 'https://platform.42crunch.com')
+    CRUNCH_API_KEY = ENV['CRUNCH_API_KEY']
+
+    class Fetch
```

Comment:
For some reason the naming of this was difficult, feel free to add better suggestions if you don't think its revealing its intention enough.

Reply:
When you say difficult, do you mean that:
- it was difficult to find a name that satisfactorily expressed the intention of the class, 
- or that the name clashed with another namespace or in some other way caused runtime issues?

Reply:
IMO it can be hard to make class names expressive, as verbose class names are awkward to use. Harder than a verbose method name. So if the name doesn't fully convey the intention of the object then comments should be used immediately before the class definition.